phosphobot/models.py

```diff
@@ -1524,6 +1679,16 @@ def save(self, meta_folder_path: str) -> None:
         """
         self.to_jsonl(meta_folder_path)
 
+    def save_writing_file(self, meta_folder_path: str) -> None:
+        """
+        Save the episodes to the meta folder path.
+        This overwrite the file instead of appending to it.
+        This is used when removing an episode from the dataset.
+        """
+        with open(f"{meta_folder_path}/episodes.jsonl", "w") as f:
+            for episode in self.episodes:
+                f.write(episode.model_dump_json() + "\n")
```

Comment:
Can you please instead add a parameter `save_mode: Literal["append", "overwrite"]` to the .save function to avoid multiplying methods